### PR TITLE
import export dashboard

### PR DESCRIPTION
**Pull request by Sumragen**

### CODE REVIEW

**Review by Olensavchenko — APPROVED** [review]

File: packages/administration/dashboards-config/widget-edit/widget-copy-dialog/widget-copy-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+};
+@Component({
+    selector: 'xm-widget-copy-dialog',
+    templateUrl: './widget-copy-dialog.component.html',
+    styleUrls: ['./widget-copy-dialog.component.scss'],
+    standalone: true,
+    imports: [
+        MatDialogModule,
+        MatButtonModule,
+        XmTranslationModule,
```
Comment (Kaskyi) [inline packages/administration/dashboards-config/widget-edit/widget-copy-dialog/widget-copy-dialog.component.ts:48]: need morre

File: packages/administration/dashboards-config/dashboards-list/dashboards-list.component.ts [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
 import { DashboardsExportService } from './dashboards-export.service';
 import { DashboardsImportService } from './dashboards-import.service';
 import { CdkDragDrop, moveItemInArray } from '@angular/cdk/drag-drop';
 import { XmTextControl, XmTextControlOptions } from '@xm-ngx/components/text';
 import { Location } from '@angular/common';
+import { readFromClipboard, takeUntilOnDestroy, takeUntilOnDestroyDestroy } from '@xm-ngx/operators';
+import { MatDialog } from '@angular/material/dialog';
+import {
+    DashboardsListCopyDialogComponent, OPERATIONS,
+} from '@xm-ngx/administration/dashboards-config/dashboards-list/dashboards-list-copy-dialog/dashboards-list-copy-dialog/dashboards-list-copy-dialog.component';
```
Comment (Kaskyi) [inline packages/administration/dashboards-config/dashboards-list/dashboards-list.component.ts:24]: make relative

File: packages/administration/dashboards-config/dashboards-list/dashboards-list-expand/dashboards-list-expand.component.ts [hunk 3]
```diff
@@ -1,17 +1,27 @@
...
 import { DashboardsManagerService } from '../../dashboards-manager.service';
 
 import { DashboardCollection, WidgetCollection } from '../../injectors';
 import { WidgetEditComponent } from '../../widget-edit/widget-edit.component';
+import { readFromClipboard, takeUntilOnDestroy, takeUntilOnDestroyDestroy } from '@xm-ngx/operators';
+import { XmToasterService } from '@xm-ngx/toaster';
+import { MatDialog } from '@angular/material/dialog';
+import {
+    OPERATIONS,
+} from '@xm-ngx/administration/dashboards-config/dashboards-list/dashboards-list-copy-dialog/dashboards-list-copy-dialog/dashboards-list-copy-dialog.component';
```
Comment (Kaskyi) [inline packages/administration/dashboards-config/dashboards-list/dashboards-list-expand/dashboards-list-expand.component.ts:21]: relative

**Review by Kaskyi — APPROVED** [review]

File: packages/administration/dashboards-config/dashboards-list/dashboards-list-expand/dashboards-list-expand.component.ts (hunk 3 above)
Comment (Olensavchenko) [reply packages/administration/dashboards-config/dashboards-list/dashboards-list-expand/dashboards-list-expand.component.ts]: done

File: packages/administration/dashboards-config/dashboards-list/dashboards-list.component.ts (hunk 2 above)
Comment (Olensavchenko) [reply packages/administration/dashboards-config/dashboards-list/dashboards-list.component.ts]: done

File: packages/administration/dashboards-config/widget-edit/widget-copy-dialog/widget-copy-dialog.component.ts (hunk 1 above)
Comment (Kaskyi) [reply packages/administration/dashboards-config/widget-edit/widget-copy-dialog/widget-copy-dialog.component.ts:48]: image
))

**Review by Bozonych — APPROVED** [review]

**Pull request merged**